doc/contributing/distribution.md

```diff
@@ -0,0 +1,17 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes many dependencies, such as V8, Undici and others; and comes
+bundled with standalone tools such as `npm`. It is not a goal of the Node.js
+project to provide a level playing field for all dependencies or bundled tools.
```

Comment:
Instead of listing non-goals, let’s say what you really mean here: npm distribution in the default installer is only justified by historical reasons, and is likely going to stay this way for the foreseeable future to avoid disturbing the ecosystem.

Reply:
I don’t know why you think that that’s what I really mean. I don’t view npm’s inclusion as some kind of regretful historical artifact. It’s the standard package manager, for no other reason than that it’s published by the people who run the de facto standard package registry, and so therefore its inclusion is a sensible default.

The point of this is so that when someone complains that it’s not fair that we ship npm, so for equity/fairness/level playing field/competition/anticompetitiveness/etc therefore we need to also ship one or more other package managers, the reason why we don’t is because our policy is to ship only one type of each category of tool. It’s a drain on finite resources to support multiple, and promoting fairness in the marketplace is not our responsibility.

Reply:
I believe this is trying to answer the question I was trying to ask in #50963 which **is** a more general question. I think that from the perspective of attempting to "provide clarity" this more general wording is better than the more direct thing even if we all know the reason. This way the document presents a more general stance of the project.

I do think that the "non-goals" framing is maybe not necessary to get the right point across. The point is to follow the [technical values](https://github.com/nodejs/node/blob/f4af4b111c20e599742223bc1a78b0a7017870a7/doc/contributing/technical-values.md) and put DX and stability first. Corepack is not the best we as a community can do for DX and knowing that, enabling it by default is not the best solution is not good for future stability. Maybe a better way to state this is:

```suggestion
It is the goal of the Node.js project to provide a great and stable developer experience, which means we cannot ship all things for all user groups.
```

Reply:
Oh this was marked resolved as I commented. @GeoffreyBooth maybe could we re-open it until the conversation is complete?

Reply:
This wasn’t inspired by Corepack, it was inspired the previous discussion around shipping Yarn: https://github.com/nodejs/TSC/issues/1012#issuecomment-820476633. I really meant what I wrote here, that in general we shouldn’t ship multiple tools for the same purpose. That’s not fair to Yarn, yes, but there are only so many Node contributors and promoting competition in the ecosystem is not one of our project goals (not that we oppose competition or fairness by any means, but it shouldn’t come at a cost of draining time and effort from the Node project itself).

Reply:
> Instead of listing non-goals, let’s say what you really mean here: npm distribution in the default installer is only justified by historical reasons, and is likely going to stay this way for the foreseeable future to avoid disturbing the ecosystem.

I would encourage you not to presume intent when the justification was explicitly written (ie. `"It is not a goal of the Node.js
project to provide a level playing field for all dependencies or bundled tools."`). There is no reference to historical precedence in this PR. 

As @GeoffreyBooth has already confirmed, this policy simply codifies that `"a level playing field"` is not criteria or justification that will be considered when determining a dependency or bundled tool's value to the project.

I'm fine with the wording. A "non-goal" is simply a "statement" & statements are required to define a "policy". If its accepted, & merged, it becomes "fact".

Reply:
That makes sense, I didn't mean to make any comments on your intent, just that it is alright if folks think (know) that this new policy is required *because* of all the other things. That doesn't change the longer term value in having clarity on this topic, nor does it mean the wording is/is not good.  think that this wording is framing as "negative" what could be phrased as a "positive".  This really is about what the project **is doing** (providing good dx and stability for the ecosystem).

Reply:
I mean, I could just delete “It is not a goal of the Node.js project to provide a level playing field for all dependencies or bundled tools.” if it bothers people. It’s arguably redundant given the following sentence.

The problem with saying that a goal of Node is to promote good DX is a) we have a different document that defines our technical values, and b) good DX is in the eye of the beholder, and there are those who would argue that a vibrant ecosystem is good DX, and therefore a goal of good DX commits us to a competitive ecosystem etc. What I _intend_ it to close off a justification for including potential additions to our distribution, and the most efficient way to phrase that (that I can think of) is to state non-goals.

Reply:
I think that sentence should be removed, listing things which are "not a goal" is a bad idea.

> I don’t know why you think that that’s what I really mean.

Sorry for misreading you then.

Reply:
> I think that sentence should be removed, listing things which are "not a goal" is a bad idea.

I explicitly disagree. When referring to a document like this, understanding what the scope is _intended_ to be used for and what it's _not intended_ to be used for is extremely valuable.

Reply:
In this case, I'd challenge that it'd be extremely valuable. Please remove it.

Reply:
Note: sorry for the duplicate, in an attempt to fix the broken GH UI from submitting the comment with a review approval my original comment was deleted. Trying to recreate it here.

I think there is a better way to get the same point across without the negative framing. Something along the lines of:

> It is the goal of the Node.js project to provide a stable UX and this means that we cannot ship all features/tools/etc for all user groups.